app/routes.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+"""
+Volcano InSAR Interpretation Workbench
+
+SPDX-License-Identifier: MIT
+
+Copyright (C) 2021-2024 Government of Canada
+
+Authors:
+  - Drew Rotheram <drew.rotheram-clarke@nrcan-rncan.gc.ca>
+"""
+from flask import Response, request
+import requests
+import boto3
+
+
+def add_routes(server):
+    def get_signed_url(bucket, key):
+        print(bucket)
+        s3_client = boto3.client('s3')
+        url = s3_client.generate_presigned_url(
+            'get_object',
+            Params={'Bucket': bucket, 'Key': key},
+            ExpiresIn=60  # URL expires in 60 seconds
+        )
+        print(url)
```

Comment:
I recommend using [logging](https://docs.python.org/3/library/logging.html) with appropriate log levels instead `print` for logging in committed code. This allows logging to configured to to different levels in development vs running in production. Reworking this function to log signed URLs with [logging.debug](https://docs.python.org/3/library/logging.html#logging-levels) in particular would be a good idea, as printing every signed URL to the console is going to produce a lot of log output that may not be desirable in deployed environments.

Reply:
Fixed in https://github.com/Volcano-Risk-Reduction-in-Canada/Volcanic-Interpretation-Workbench/pull/68/commits/fba0ae92ef3d776e89cba5fdf3fbc3cdb5211adf

Logging level for these set to INFO